examples/stable-video-diffusion/README.md

```diff
@@ -0,0 +1,61 @@
+<!---
+Copyright 2024 The HuggingFace Team. All rights reserved.
```

Comment:
can we move the full thing to stable diffusion folder?

Reply:
Done